gobblin-modules/gobblin-kafka-09/src/test/java/org/apache/gobblin/metrics/reporter/KafkaKeyValueProducerPusherTest.java

```diff
@@ -42,6 +42,7 @@
 /**
  * Test {@link KafkaKeyValueProducerPusher}.
  */
+@Test (groups = {"disabledOnTravis"})
```

Comment:
Let's remove this, since this test is not the root cause of travis build failures.